cfme/tests/automate/__init__.py

```diff
@@ -1 +1,67 @@
 # -*- coding: utf-8 -*-
+from textwrap import dedent
+
+script01 = dedent(
```

Comment:
Q: Is there any descriptive name to give to this script?

Reply:
Yes. We can give name. I have updated with this one: ```user_list_hash_data```. Because this method creates user list and shows this hash data in simulation tree or in automation log